src/main/java/com/icegreen/greenmail/imap/commands/CreateCommand.java

```diff
@@ -27,7 +31,7 @@ protected void doProcess(ImapRequestLineReader request,
                              ImapSession session)
             throws ProtocolException, FolderException, AuthorizationException {
         String mailboxName = parser.mailbox(request);
-        parser.endLine(request);
+//        parser.endLine(request);

```

Comment:
ok thanks, it just looks weird to have this line commented.
Don't you prefer to either remove it at all or tell in a comment why you did that ?